src/main/java/pers/gwyog/gtneioreplugin/util/GT5OreLayerHelper.java

```diff
@@ -12,11 +14,28 @@
 
 public class GT5OreLayerHelper {
 
+    class oreDimensionWrapper {
```

Comment:
1) Per convention, Java class names always start with an uppercase letter.
2) Please declare the class as private, if you don't intend it to be used outside of `GT5OreLayerHelper`.